### ISSUE TITLE
[ISSUE for v2]: Outputted voice echoes

### ISSUE THREAD

**Issue by Aadit420** [w-okada/voice-changer#1370]
### Voice Changer Version

vcclient_win_cuda_2.0.61-alpha.zip

### Operational System

Windows 11

### GPU

Nvidia RTX 3060 Mobile

### CUDA Version

cuda_12.6.r12.6/compiler.34841621_0

### Read carefully and check the options

- [X] If you use win_cuda_torch_cuda edition, setup cuda? [see here](https://onnxruntime.ai/docs/execution-providers/CUDA-ExecutionProvider.html#requirements)
- [X] If you use win_cuda edition, setup cuda and cudnn? [see here](https://onnxruntime.ai/docs/execution-providers/CUDA-ExecutionProvider.html#requirements)
- [ ] If you use mac edition, client is not launched automatically. Use chrome to open application.?
- [X] I've tried to change the Chunk Size
- [X] I've tried to set the Index to zero
- [X] I've read the [tutorial](https://github.com/w-okada/voice-changer/blob/master/tutorials/tutorial_rvc_en_latest.md)
- [X] I've tried to extract to another folder (or re-extract) the .zip file

### Does pre-installed model work?

No

### Model Type

RVC

### Issue Description

The outputted voice echoes. I've tried setting my sound source to Server mode and change APIs but the application would crash with something like "mismatched sampling rate".
Sadly, I can't send a .wav file here to show you what I mean by the first part, but the ending is as if it's chattering to itself. I'm not sure how to describe it. 

This does not happen in the Google colab version, wich is even more strange

### Application Screenshot

![image](https://github.com/user-attachments/assets/b2af1633-8eba-4cad-adea-dd47cd1ba79b)

### Logs on console

G:\vcclient_win_cuda_2.0.61-alpha>main.exe cui --https true --no_cui True
checking the modules...

                          vcclient

--------------------------------------------------------------
 Application | https://192.168.1.19:18000
 Log(rich)   | https://192.168.1.19:18000/?app_mode=LogViewer
 Log(text)   | https://192.168.1.19:18000/vcclient.log
 API         | https://192.168.1.19:18000/docs
 License(js) | https://192.168.1.19:18000/licenses-js.json
 License(py) | https://192.168.1.19:18000/licenses-py.json
--------------------------------------------------------------
Please press Ctrl+C once to exit vcclient.

**Comment by Kuuko-fokkusugaru** [comment]
Of course, if you set the input device to be your system, it will listen to every sound that your pc does. So everything being out through monitor will also be listened by the system input and create a loop.
You need to set a microphone in the input field, not your windows system sounds.

**Comment by Aadit420** [comment]
So there's no way for it to not loop with itself? That's a shame, especially when this issue did not happen with google colab under the exact same conditions

**Comment by Kuuko-fokkusugaru** [comment]
The way to not loop it's to not select the system sound or to simply disable monitor. It's there any particular reason for you to select system sound instead of your microphone? If you want to convert audio other than from a microphone, there is a file option which allows you to select an audio file to play and convert.
You can also consider just disabling the monitor setting it to none. The converted audio will still go through the VAC output just fine, you will simply not hear the preview which will avoid getting it again in a loop.
Basically, the way that it's acting up is expected because your configuration is not the right one.

**Comment by Aadit420** [comment]
I see. Well, setting the monitor to none doesn't help but what does is switching input to Stereo mix.
I wanted to use this as a way to testing how the models sound without having to use my microphone (I live in a very close place with family members and don't want to disturb them)
With the input in Stereo Mode, it seems no echoing is present
